pixeltable/tests/io/test_pandas.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+
+import numpy as np
+
+import pixeltable as pxt
+from ..utils import skip_test_if_not_installed
+
+
+class TestPandas:
+
+    def test_pandas_csv(self, reset_db) -> None:
+        from pixeltable.io import import_csv
```

Comment:
Are we really opposed to having those functions be visible in the top-level namespace?